Libplanet.Tests/Assets/CurrencyTest.cs

```diff
@@ -255,5 +255,39 @@ public void Serialize()
 
             Assert.Equal(bar, new Currency(bar.Serialize()));
         }
+
+        [Fact]
+        public void JsonSerialization()
+        {
+#pragma warning disable CS0618  // must test obsoleted Currency.Legacy() for backwards compatibility
+            var foo = Currency.Legacy("FOO", 2,  null);
```

Comment:
You can pass this part, but there was double spaces.

```suggestion
            var foo = Currency.Legacy("FOO", 2, null);
```